Packs/Oletools/Scripts/OletoolsScript/OletoolsScript.py

```diff
@@ -0,0 +1,202 @@
+        args.append(command)
+        args.append(file)
+
+        output = subprocess.run(args, capture_output=True)
+
+        regex = r"Found relationship 'hyperlink' with external link (.*?)\n"
+        str_output = output.stdout.decode("utf-8")
+        matches = re.findall(regex, str_output, re.MULTILINE)
+        readable_md = '### Found the following relationship "hyperlink" with external links\n'
+        hyperlink_list = []
+
+        if not matches:
+            readable_md = '### No "hyperlink" with external links were found'
+        else:
+            for match in matches:
+                readable_md += f'- {match}\n'
+                hyperlink_list.append(match)
+
+        cr = CommandResults(readable_output=readable_md, outputs_prefix='Oletools.Oleobj',
+                            outputs={'hyperlinkes': hyperlink_list},
+                            raw_response=str_output)
+        return cr
+
+    def olevba(self):
+        file_data = open(self.processed_file_path, 'rb').read()
+        vbaparser = VBA_Parser(self.processed_file_path, data=file_data, disable_pcode=True)
+
+        if not vbaparser.detect_vba_macros():
+            return CommandResults(readable_output='### No VBA Macros found\n')
+
+        found = '### VBA Macros found\n'
+        all_macros = vbaparser.extract_all_macros()
+        macros_list = []
+
+        for macro in all_macros:
+            macros_list.append({
+                'VBA Macro': macro[2],
+                'Found in file': macro[0],
+                'Ole stream': macro[1]
+            })
+
+        macros_list_md = tableToMarkdown('Macros found', macros_list,
+                                         headers=['VBA Macro', 'Found in file', 'Ole stream'])
+
+        macro_source_code = vbaparser.reveal()
+        readable_macro = f'\n### Macro source code\n {macro_source_code}\n'
+
+        results = vbaparser.analyze_macros(show_decoded_strings=self.show_decoded)
+        results_list = []
+        for result in results:
+            results_list.append({
+                'Type': result[0],
+                'Keyword': result[1],
+                'Description': result[2]
+            })
+
+        results_md = tableToMarkdown('Macro Analyze', results_list, headers=['Type', 'Keyword', 'Description'])
+        vbaparser.close()
+
+        readable_output = found + macros_list_md + readable_macro + results_md
+        outputs = {
+            'macro_list': macros_list,
+            'macro_src_code': macro_source_code,
+            'macro_analyze': results_list
+        }
+        cr = CommandResults(readable_output=readable_output, outputs_prefix='Oletools.Olevba', outputs=outputs)
+        return cr
+
+
+def main():  # pragma: no cover
+    args = demisto.args()
+    ole_command = args.get('ole_command')
+    attach_id = args.get('entryID', '')
+    file_info = demisto.getFilePath(attach_id)
+    show_decoded = args.get('decode', False)
+    password = argToList(args.get('password', ''))
```

Comment:
This argument is passed to the function python function as a list. maybe its a better practice to get it in a regular way and then rap in the .py but actually there can be an option to give a list of potential passwords to test and it can work so maybe we can change it also in the yml and allow list i will discuss with @tomer-pan

Reply:
Done